api/app/main.py

```diff
@@ -161,19 +163,49 @@ def stats(stats_model: StatsModel) -> list[dict[str, Any]]:
 
 
 @app.get("/acled")
-def get_acled_incidents(request: Request):
+def get_acled_incidents(
+    iso: int,
+    limit: int,
+    fields: Optional[str],
+    email: str,
+    key: str,
+    event_date: Optional[date],
+):
     acled_url = "https://api.acleddata.com/acled/read"
 
     try:
```

Comment:
I don't think you need this try/except anymore since the typing you introduced should take care of this now.